src/metabase/lib/page.clj

```diff
@@ -0,0 +1,25 @@
+(ns metabase.lib.page
+  (:require
+   [metabase.lib.schema :as lib.schema]
+   [metabase.lib.util :as lib.util]
+   [metabase.util :as u]
+   [metabase.util.malli :as mu]))
+
+(mu/defn current-page :- [:maybe ::lib.schema/page]
```

Comment:
we surprisingly had no Lib functions related to `:page`... which makes me think that since we actually don't even support it on the FE maybe we should drop it from MBQL??